src/main/java/chess/domain/ChessGame.java

```diff
@@ -5,18 +5,19 @@
 import chess.domain.position.Square;
 
 public final class ChessGame {
-    private static final String ERROR_MESSAGE_TURN = "[ERROR] 순서 지키시지?!";
+    private static final String ERROR_MESSAGE_TURN = "순서 지키시지?!";
 
     private Board board;
     private GameTurn turn;
 
-    public ChessGame(BoardGenerator boardGenerator) {
+    public ChessGame(BoardGenerator boardGenerator, GameTurn gameTurn) {
         this.board = new Board(boardGenerator);
-        this.turn = GameTurn.READY;
+        this.turn = gameTurn;
     }
 
     public void startGame() {
-        turn = GameTurn.WHITE;
+        if (turn.equals(GameTurn.READY))
+        this.turn = GameTurn.WHITE;
```

Comment:
```suggestion
        if (turn.equals(GameTurn.READY)) this.turn = GameTurn.WHITE;
```
중괄호가 없다면 한줄에 표현하는게 좀 더 명확한 것 같고, 중괄호를 넣어주는게 가장 명확할 것 같기는 해요.

개인적으로는 ready일 때만 호출되는 메서드인데 이렇게 따로 분리할 필요가 있나라는 생각이 들기는 했습니다~

Reply:
앗 중괄호가 없는지 전혀 몰랐네요...! 🤣

저도 이 부분은 리팩터링하면서 ready 일경우 무조건 white 로 turn 을 변경해주는데, 굳이 분리해야 할 메서드인지 고민을 했습니다. 

해당 메서드는 없애는 방향으로 리팩터링 해보겠습니다 :)